sycl/include/CL/sycl/group_algorithm.hpp

```diff
@@ -20,6 +22,8 @@
 #include <sycl/ext/oneapi/experimental/group_sort.hpp>
 #include <sycl/ext/oneapi/functional.hpp>
 
+#define SYCL_EXT_ONEAPI_COMPLEX_ALGORITHMS 1
```

Comment:
Can we have this definition in https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/feature_test.hpp.in like most other extensions?

Reply:
Thanks!  And I updated the tests as well.